docs/deployment/docker-setup.md

```diff
@@ -1,9 +1,13 @@
 # Docker
-## What is docker
-TODO
+## Docker in really short
+In really short, docker is like a virtual machine but without the kernel emulation. Every container share the host kernel. For rtc, we use linux containers that require a linux kernel.
+
+On Windows or Mac, linux containers run inside a virtual machine. This Virtual machine provides the linux kernel. On linux, it's the host kernel that is shared with the containers.
```

Comment:
May be better: 'On Windows or Mac, Linux containers run inside virtual machine. This virtual machine...'

Reply:
In next commit